<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>1121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EPLab</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QDockWidget" name="scoreDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Степень различия</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="score_label">
       <property name="minimumSize">
        <size>
         <width>182</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt; color:#73d216;&quot;&gt;0%&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="freqDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Частота пробного сигнала</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QRadioButton" name="frequency_10hz_radio_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>10 Гц</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="frequency_100hz_radio_button">
       <property name="text">
        <string>100 Гц</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="frequency_1khz_radio_button">
       <property name="text">
        <string>1 кГц</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="frequency_10khz_radio_button">
       <property name="text">
        <string>10 кГц</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="frequency_100khz_radio_button">
       <property name="text">
        <string>100 кГц</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="currentDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Чувствительность по току</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QRadioButton" name="sens_low_radio_button">
       <property name="text">
        <string>Низкая</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="sens_medium_radio_button">
       <property name="text">
        <string>Средняя</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="sens_high_radio_button">
       <property name="text">
        <string>Высокая</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>400</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="voltageDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Амплитуда пробного сигнала</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QRadioButton" name="voltage_1_2v_radio_button">
       <property name="text">
        <string>1.2 В</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="voltage_3_3v_radio_button">
       <property name="text">
        <string>3.3 В</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="voltage_5v_radio_button">
       <property name="text">
        <string>5 В</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="voltage_12v_radio_button">
       <property name="text">
        <string>12 В</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="planDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Режимы</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QLineEdit" name="line_comment_pin">
       <property name="placeholderText">
        <string>Комментарий...</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_comment_push_button">
       <property name="text">
        <string>Сохранить комментарий</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="test_plan_tab_widget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="test_plan_tab_S">
        <attribute name="title">
         <string>Сравнение</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="ivc_state_group_box">
             <property name="title">
              <string>ВАХ</string>
             </property>
             <layout class="QVBoxLayout" name="ivc_state_group_box_layout">
              <item>
               <widget class="QCheckBox" name="freeze_curve_a_check_box">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 255);</string>
                </property>
                <property name="text">
                 <string>Фиксировать A</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="freeze_curve_b_check_box">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Фиксировать B</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="image_group_box">
             <property name="title">
              <string>Изображение</string>
             </property>
             <layout class="QHBoxLayout" name="save_image_layout">
              <item>
               <widget class="QPushButton" name="save_image_push_button">
                <property name="font">
                 <font>
                  <pointsize>7</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Сохранить изображение</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="test_plan_tab_ZP">
        <attribute name="title">
         <string>Запись</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="zp_push_button_left">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>&lt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="zp_label_num">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="zp_push_button_right">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="zp_push_button_save">
               <property name="text">
                <string>Сохранить</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="zp_push_button_new_point">
               <property name="text">
                <string>Новая точка</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="zp_new_file_button">
           <property name="text">
            <string>Создать пустой план</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="zp_save_file_button">
           <property name="text">
            <string>Сохранить план</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="zp_open_file_button">
           <property name="text">
            <string>Открыть план</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="zp_add_image_button">
           <property name="text">
            <string>Добавить изображение</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="zp_save_file_as_button">
           <property name="text">
            <string>Сохранить план как ...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="test_plan_tab_TP">
        <attribute name="title">
         <string>Тестирование</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="tp_push_button_left">
               <property name="text">
                <string>&lt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="tp_label_num">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="tp_push_button_right">
               <property name="text">
                <string>&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="tp_push_button_save">
             <property name="text">
              <string>Сохранить изображение</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4"/>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="tp_open_file_button">
           <property name="text">
            <string>Открыть файл</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu_action">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="new_file_action"/>
    <addaction name="open_file_action"/>
    <addaction name="save_file_action"/>
    <addaction name="save_as_file_action"/>
    <addaction name="separator"/>
    <addaction name="save_screen_action"/>
   </widget>
   <widget class="QMenu" name="view_menu_action">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="open_window_board_action"/>
    <addaction name="freeze_curve_a_action"/>
    <addaction name="freeze_curve_b_action"/>
   </widget>
   <widget class="QMenu" name="mode_menu_action">
    <property name="title">
     <string>Режим</string>
    </property>
    <addaction name="comparing_mode_action"/>
    <addaction name="writing_mode_action"/>
    <addaction name="testing_mode_action"/>
    <addaction name="separator"/>
    <addaction name="sound_enabled_action"/>
    <addaction name="separator"/>
    <addaction name="settings_mode_action"/>
   </widget>
   <widget class="QMenu" name="help_menu_action">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="about_action"/>
   </widget>
   <widget class="QMenu" name="test_plan_menu_action">
    <property name="title">
     <string>План тестирования</string>
    </property>
    <addaction name="next_point_action"/>
    <addaction name="last_point_action"/>
    <addaction name="separator"/>
    <addaction name="new_point_action"/>
    <addaction name="save_point_action"/>
    <addaction name="add_board_image_action"/>
   </widget>
   <addaction name="file_menu_action"/>
   <addaction name="view_menu_action"/>
   <addaction name="mode_menu_action"/>
   <addaction name="test_plan_menu_action"/>
   <addaction name="help_menu_action"/>
  </widget>
  <action name="new_file_action">
   <property name="text">
    <string>Создать план тестирования</string>
   </property>
  </action>
  <action name="open_file_action">
   <property name="text">
    <string>Открыть план тестирования</string>
   </property>
  </action>
  <action name="save_file_action">
   <property name="text">
    <string>Сохранить план тестирования</string>
   </property>
  </action>
  <action name="save_as_file_action">
   <property name="text">
    <string>Сохранить план тестирования как</string>
   </property>
  </action>
  <action name="save_screen_action">
   <property name="text">
    <string>Сохранить скриншот</string>
   </property>
  </action>
  <action name="open_window_board_action">
   <property name="text">
    <string>Открыть окно просмотра платы</string>
   </property>
  </action>
  <action name="freeze_curve_a_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Заморозить канал А</string>
   </property>
  </action>
  <action name="freeze_curve_b_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Заморозить канал B</string>
   </property>
  </action>
  <action name="comparing_mode_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Сравнение</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="writing_mode_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Запись плана тестирования</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="testing_mode_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Тестирование по плану</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="setting_mode_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Настройка</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="sound_enabled_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Звук</string>
   </property>
  </action>
  <action name="settings_mode_action">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="next_point_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Следующая точка</string>
   </property>
  </action>
  <action name="last_point_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Предыдущая точка</string>
   </property>
  </action>
  <action name="new_point_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Новая точка</string>
   </property>
  </action>
  <action name="save_point_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить точку</string>
   </property>
  </action>
  <action name="add_board_image_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Добавить изображение</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
